### PR TITLE
feat(shelter): new computation for windows area #72

### LINKED ISSUE #72: [FEATURE] Geometry page - improve graphics
Add labels/names for each geometry:

1. Double pitch (gable) roof
2. Single pitch roof
3. Dome
4. Irregular 
5. Conical roof

Improve graphic for irregular form.

### PR DESCRIPTION
**Pull request by guilbep**
Closes #72

# Describe your changes

- Replace Openings area by Windows area only
- Remove Watch that were triggering infinite loops

# Related GitHub issues and pull requests

- Ref: #72

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] Don't forget to link PR to issue if you are solving one.

### CODE REVIEW

File: frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step2Geometry.vue [hunk 1]
```diff
@@ -16,193 +16,214 @@
...
-              <v-btn icon @click.stop="selectedItem = geometry">
-                <v-icon>{{ mdiMagnify }}</v-icon>
-              </v-btn>
-              <!-- <span>{{ geometry._id }}</span> -->
-            </v-card-title>
-            <v-card-text
-              v-if="!geometry.hiddenInputs && shelter_geometry_type"
-              class="flex-gap-sm d-flex flex-column justify-space-between"
+      <v-col>
+        <v-container>
```
Comment (blueur) [inline frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step2Geometry.vue:20]: un seul v-container à la racine, puis que des row et col.

File: frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step2Geometry.vue [hunk 2]
```diff
@@ -515,6 +511,27 @@ export default class Step2Geometry extends Vue {
...
+    {
+      _id: "dome",
+      name: "Dome - Circle or ellipse plan",
+      image_url: "/houses_new/SSC_P4_Shelter-Dome_220201.png",
+      shelter_dimensions: ["L", "W", "H"],
+      door_dimensions: ["Wd", "Hd"],
+      window_dimensions: ["Ww", "Hw", "Hs"],
+      volumeFunction(shelterDimension: ShelterDimensions): number {
+        const { L, W, H } = shelterDimension || {};
+        if (!L || !W || !H) {
```
Comment (blueur) [inline frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step2Geometry.vue:523]: if shelterDimension is defined ?

File: frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step2Geometry.vue [hunk 3]
```diff
@@ -364,8 +369,20 @@ export default class Step2Geometry extends Vue {
...
+    this.localShelter.technical_performance.input_2a_1 =
+      windowArea / floorArea > 0.05 ? 1 : undefined;
+
+    // Ratio of windows area to floor area > 0.10
+    this.localShelter.technical_performance.input_2c_1 =
+      windowArea / floorArea > 0.1 ? 1 : undefined;
+  }
 
   private doorDimensions(doorDimensions: DoorDimensions[]): number {
+    // deprecated. We were using it for Opening area, but it's unused for now
```
Comment (blueur) [inline frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step2Geometry.vue:385]: @deprecated ?

File: frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step2Geometry.vue [hunk 4]
```diff
@@ -364,8 +369,20 @@ export default class Step2Geometry extends Vue {
       };
     }
   }
+  private updateTechnicalPerformance(): void {
+    const { windowArea, floorArea } = this.localShelter.geometry;
+    // update technical_performance, because it should be done like this (defined in the specs)
+    // Ratio of window and ventilation openings area to floor area > 0.05
+    this.localShelter.technical_performance.input_2a_1 =
+      windowArea / floorArea > 0.05 ? 1 : undefined;
```
Comment (blueur) [inline frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step2Geometry.vue:377]: 1 et undefined sont les uniques valeurs possibles ?

Comment (guilbep) [reply frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step2Geometry.vue:377]: Oui

**Comment by guilbep** [comment]
Merged by merging #81 in release v0.3.0

File: frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step2Geometry.vue (hunk 2 above)
Comment (guilbep) [reply frontend/src/views/shelter_sustainability/ShelterSustainabilityItem/Step2Geometry.vue:523]: @blueur tu l'écrirais comment ça toi ?

**Pull request closed without merging**